CITATION.cff

```diff
@@ -3,7 +3,7 @@ title: "GLASS: Generator for Large Scale Structure"
 message: >-
   If you use GLASS in your research, please cite the associated paper.
 repository-code: "https://github.com/glass-dev/glass"
-url: "https://glass.readthedocs.io"
+url: "https://glass.readthedocs.io/stable"
```

Comment:
This can remain at the top-level URL since it forwards to `/stable`.

Reply:
I still think we should have it anyway, it's good practice. Is there a good reason to not?

Reply:
As a safety net. Surely it follows a similar argument to https://github.com/glass-dev/glass/pull/416#discussion_r1843685913?

Reply:
Thinking along the same lines, I think "the URL" is the root one which is always safe to link to a valid place (e.g., when /en was removed). Below we need to point specifically to "the docs for stable/v2024.2" so I don't think the same argument applies.

Reply:
And for `pyproject.toml`